### PR TITLE
refactor: helm chart

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: helm/templates/_resources.tpl [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+{{- define "route.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "route" | indent 1 }}
+{{- end }}
+
+{{/*
+Create ingress name as used by the service name label.
+*/}}
+{{- define "ingress.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "ingress" | indent 1 }}
+{{- end }}
```
Comment (asafMasa) [inline helm/templates/_resources.tpl]: end of file

File: helm/templates/deployment.yaml [hunk 2]
```diff
@@ -35,16 +38,19 @@ spec:
...
         release: {{ $releaseName }}
         run: {{ $releaseName }}-{{ $chartName }}
-        {{- if .Values.rasterCommon.useNodeAffinity }}
+        {{- if .Values.global.useNodeAffinity }}
         nodeAffinity: nodeAffinityPod
         {{- end }}
         {{- include "discrete-overseer-creator.selectorLabels" . | nindent 8 }}
-      {{- if .Values.resetOnConfigChange }}
       annotations:
+        {{- if .Values.resetOnConfigChange }}
```
Comment (asafMasa) [inline helm/templates/deployment.yaml:46]: where does it end?

File: helm/templates/deployment.yaml [hunk 3]
```diff
@@ -58,33 +64,33 @@ spec:
...
-              value: {{ printf "%s/%s" .Values.rasterCommon.ca.path .Values.rasterCommon.ca.key | quote }}
+              value: {{ printf "%s/%s" .Values.global.ca.path .Values.global.ca.key | quote }}
             - name: NODE_EXTRA_CA_CERTS
-              value: {{ printf "%s/%s" .Values.rasterCommon.ca.path .Values.rasterCommon.ca.key | quote }}
+              value: {{ printf "%s/%s" .Values.global.ca.path .Values.global.ca.key | quote }}
             {{- end }}
           envFrom:
             - configMapRef:
-                name: {{ $releaseName }}-configmap
+                name: {{ $configmapName  }}
```
Comment (asafMasa) [inline helm/templates/deployment.yaml]: remove extra space

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: helm/templates/_resources.tpl (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply helm/templates/_resources.tpl]: fixed

**Review by asafMasa — APPROVED** [review]

**Pull request merged**